### ISSUE TITLE
Error updating to Meteor 1.3.4.4

### ISSUE THREAD

**Issue by MichaelJCole** [meteorhacks/kadira#246]
Hi, I'm getting this error when updating:

```
☔  meteor update                                                      A1Mod dev
This project is already at Meteor 1.3.4.4, the latest release.
=> Errors while upgrading packages:           
                                              
While checking for meteorhacks:kadira-binary-deps@1.5.0:
error: No compatible binary build found for this package. Contact the package
author and ask them to publish it for your platform.
```

**Comment by logankoester** [comment]
Yep, same here. :+1:

**Comment by benlavalley** [comment]
You can get around this issue temporarily by just downloading the package locally. I am sure Arunoda will have the issue fixed soon as it appears to be an issue with how the package is defined in Atmosphere.

cd <meteor project> && mkdir packages && cd packages && git clone https://github.com/meteorhacks/kadira-binary-deps.git && cd .. && meteor update

**Comment by logankoester** [comment]
@benlavalley Thanks, I assume you just pulled it into `packages/`?

**Comment by benlavalley** [comment]
Yes, that is all that I did. The binaries  for the v8-profiler the package uses then appear to be built just fine when meteor loads the package.

**Comment by javascriptlove** [comment]
+1 same here when building on linux

**Comment by arunoda** [comment]
Sorry about the issue guys.
This build is for meteor 1.4. I thought meteor will look for an older version for current versions.

Anyway, I built new binary builds. Check now.

**Comment by AlexCollin** [comment]
Works well on Ubuntu Linux Centos, AWS, Mac OSX under meteor 1.4
Thank you
